test/CMakeLists.txt

```diff
@@ -1708,6 +1714,24 @@ if(iodaconv_bufr_query_ENABLED)
                             rrfs.t06z.adpsfc.prepbufr.nc ${IODA_CONV_COMP_TOL_ZERO}
                     DEPENDS bufr2ioda.x )
 
+  ecbuild_add_test( TARGET  test_iodaconv_bufr_ncep_rtma_aircft
+                    TYPE    SCRIPT
+                    COMMAND bash
+                    ARGS    ${CMAKE_BINARY_DIR}/bin/iodaconv_comp.sh
+                            netcdf
+                            "${CMAKE_BINARY_DIR}/bin/bufr2ioda.x testinput/bufr_ncep_rtma_aircft.yaml"
+                            rtma_ru.t0000z.aircft.tm00_nc004006.nc ${IODA_CONV_COMP_TOL_ZERO}
+                    DEPENDS bufr2ioda.x )
+
+  ecbuild_add_test( TARGET  test_iodaconv_bufr_ncep_rtma_aircft_NC004103
+                    TYPE    SCRIPT
+                    COMMAND bash
+                    ARGS    ${CMAKE_BINARY_DIR}/bin/iodaconv_comp.sh
+                            netcdf
+                            "${CMAKE_BINARY_DIR}/bin/bufr2ioda.x testinput/bufr_ncep_rtma_aircft_NC004103.yaml"
+                            rtma_ru.t0000z.aircft.tm00_nc004103.nc ${IODA_CONV_COMP_TOL_ZERO}
```

Comment:
did you want to do a little more renaming to make this a bit more descriptive:
`rmta_ru.20230610T00Z.aircft.tm00_nc004103.nc`

just a preference to have some indication by file name when the data may be valid

Reply:
> did you want to do a little more renaming to make this a bit more descriptive: `rmta_ru.20230610T00Z.aircft.tm00_nc004103.nc`
> 
> just a preference to have some indication by file name when the data may be valid

Reply:
@BenjaminRuston what you suggest? The file name in tank is - rtma_ru.t00z.aircft.tm00.nc. I added the date and subset name to this file. I think it is ok.

Reply:
think you may have mistyped the tank file I presume is `bufr` was just saying we could break slightly with convention and add more time information to the `t0000z` part instead using `20230610t0000z` 

or something like that.  presume the bufrtanks have directories with the date information.  anyway not critical, and unless Emily or Andrew think it's a fantastic idea we can just proceed